scripts/test/package-command-test.ps1

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+. "$PSScriptRoot\..\common\_common.ps1"
+
+# Restore and compile the test app
+dotnet restore "$RepoRoot\test\PackagedCommands\Commands" --runtime "osx.10.10-x64" --runtime "ubuntu.14.04-x64" --runtime "win7-x64"
```

Comment:
does it make sense to use "_"? Also will this "dotnet" always run the right "dotnet"? What if you have one in your local path already?